modules/swagger-codegen/src/main/resources/Java/apiInvoker.mustache

```diff
@@ -106,12 +106,12 @@ public class ApiInvoker {
 
     Builder builder = client.resource(host + path + querystring).accept("application/json");
     for(String key : headerParams.keySet()) {
-      builder.header(key, headerParams.get(key));
+      builder = builder.header(key, headerParams.get(key));
```

Comment:
I do agree that it appears as though it should not be needed, but it was definitely needed in our case.  With no other changes than this reassignment, our code started working.